client/src/Movies/MovieList.js

```diff
@@ -23,32 +25,12 @@ export default class MovieList extends Component {
   render() {
     return (
       <div className="movie-list">
-        {this.state.movies.map(movie => (
-          <MovieDetails key={movie.id} movie={movie} />
+      {this.state.movies.map(movie => (
+          <MovieCard key={movie.id} movie={movie} />
```

Comment:
Great work abstracting this out -- it shows you're getting the hang of React. I know it might still feel like shaky ground, but repetition will help with that.